src/couplatis/config.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+from pathlib import Path
+
+
+BASE_DIR = Path(".").resolve()
+
+
+class Config:
+    """Config"""
+    epoch = 1
+    batchsize = 256
+    num_workers = 0
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+    def __repr__(self) -> str:
+        """repr"""
+        return f"<Config epoch={self.epoch}, batch_size={self.batchsize},
```

Comment:
Have you tried to run your code locally? This will cuase a syntax error. Python codes cannot parse unterminated f-string in a line, your code end the double quote at next line. Perhaps add a `\` at this line instead of just give the code a line break.